iml-sfa/src/main.rs

```diff
@@ -357,3 +357,55 @@ impl SfaClassExt for Client {
         Ok(ys)
     }
 }
+
+#[cfg(test)]
```

Comment:
This belongs in the retry-future crate instead of here.

Reply:
Since it's testing the policy related to the fetch in this crate I don't think it makes sense to move it. I'll just remove the tests.